Provider login and course admin area

meomancer opened the pull request:
Initializing for this work
@cchristelis 
it seems that we already have official

i think we could use it and link it to provider as an user of provider
and i think we could make page-like-admin to provide provider feature for handling courses

this fix #164

dimasciput committed:
Update docker build command

dimasciput committed:
Fix travis error (#2)

meomancer committed:
Merge https://github.com/kartoza/feti into develop

meomancer committed:
Merge branch 'develop' of https://github.com/kartoza/feti into develop

cchristelis commented on django_project/feti/models/address.py:
```diff
@@ -72,4 +72,4 @@ def address_line(self):
 
     class Meta:
         app_label = 'feti'
-        verbose_name_plural = 'Addreses'
```
let's change this to to 'Addresses' rather than removing it?

cchristelis commented on django_project/feti/models/campus.py:
```diff
@@ -19,7 +19,7 @@
 class Campus(models.Model):
     """A campus where a set of courses are offered."""
     id = models.AutoField(primary_key=True)
-    campus = models.CharField('Provider', max_length=150, blank=True, null=True)
```
Why did you change this?

cchristelis commented on django_project/feti/models/campus.py:
```diff
@@ -50,7 +50,7 @@ class Campus(models.Model):
     class Meta:
         app_label = 'feti'
         ordering = ['campus']
-        verbose_name = 'Provider'
```
Let's leave this as provider. There was a big debate about this the last round, but it turns out `Provider` is the most correct name here.

cchristelis commented:
@meomancer, this is a good start. Let's keep Campus as `Provider`.

meomancer commented on django_project/user_profile/models/official.py:
```diff
@@ -13,3 +15,16 @@ class Official(models.Model):
     phone = models.CharField(
         validators=[phone_regex], blank=True, max_length=15
     )
+    provider = models.OneToOneField(Provider, related_name='official_provider')
```
@cchristelis 
as per discussion, we give just allowed form for provider
is my approach right? 

my idea is this user is representation of provider, for login and logout
and based on this user, we could give form for this provider

meomancer commented:
@cchristelis 
I reverted models verbose name

updated :
- added profile page. In there i show what provider he is, and show list of course. In there, user can update course for that campus
- list just shown for his/her campus. if he doen't has provider, nothing will be shown

_note_:
- style is still minimalize, as well as text

![screenshot from 2016-08-09 16 44 31](https://cloud.githubusercontent.com/assets/4530905/17512147/b056d028-5e50-11e6-9b70-0da7357aa2d2.png)
![screenshot from 2016-08-09 16 44 42](https://cloud.githubusercontent.com/assets/4530905/17512148/b05fb2ba-5e50-11e6-8bd8-282e52d5d13d.png)

cchristelis commented:
@meomancer, while we are still busy with the project setup, let's merge this.

meomancer committed:
added area for campus course assign
